src/main/java/com/cs50/FinanceApp/model/Purchases.java

```diff
@@ -10,23 +10,30 @@ public class Purchases {
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long iD;
 
+    @Column(name = "user_id", insertable = false, updatable = false)
     private Integer userId;
```

Comment:
tą linijke razem z adnotacją można wywalić, automatycznie jest to ogarnięte przez:
```java
    @ManyToOne
    @JoinColumn(name = "user_id")
    private User user;
```